src/Controller/AboutController.php

```diff
@@ -270,11 +270,11 @@ public function peopleAction(Request $request, string $type) : Response
                 $editorInChief = $leadership->filter(function (Person $person) {
                     return 'Editor-in-Chief' === $person->getTypeLabel();
                 });
-                $foundingEditorInChief = $leadership->filter(function (Person $person) {
+                $foundingEditorInChief = $people->filter(function (Person $person) {
```

Comment:
Hmm, this will cycle through _all_ people. Since this is all a bit of a hack, I'm more comfortable with hard-coding his ID and doing an individual request for that.

Reply:
ok, will take that approach.